src/modules/apps/exec.ts

```diff
@@ -0,0 +1,53 @@
+import * as WebSocket from 'ws'
+import * as url from 'url'
+import { getAccount, getToken, getWorkspace } from '../../conf'
+import { getManifest } from '../../manifest'
+import { region } from '../../env'
+import { toMajorRange } from '../../locator'
+const EOT = '\x04'
+export default async (command: string, options) => {
+  const manifest = await getManifest()
+  const { name, vendor, version, builders } = manifest
+  const { dotnet, node, 'service-js': serviceJs } = builders
+  if (!dotnet && !node && !serviceJs) {
+    return
+  }
+  const majorRange = toMajorRange(version)
+  const host = `${name}.${vendor}.${region()}.vtex.io`
+  const path = `/${getAccount()}/${getWorkspace()}/_exec`
+  const clientOptions = {
+    headers: {
+      Authorization: getToken(),
+      Host: host,
+      'X-Vtex-Runtime-Api': 'true',
+    },
+  }
+  var urlObject = {
+    protocol: 'ws',
+    hostname: host,
+    pathname: path,
+    query: {
+      __v: majorRange,
+      params: command?.split(' '),
+      interactive: options.i || options.interactive,
+    },
+  }
+  const formattedUrl = url.format(urlObject)
+  const ws = new WebSocket(formattedUrl, clientOptions)
```

Comment:
What do you think on this `WebSocket` being a client on the `clients` folder? I don't know if this would make sense, but would facilitate finding all toolbelt's external communications